package/boot/uboot-rockchip/src/arch/arm/dts/rk3568-mrkaio-m68s.dts

```diff
@@ -0,0 +1,268 @@
+// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
```

Comment:
Should be consistent with the kernel one

Reply:
I meant, this DTS should be consistent with the kernel one.

\+ RNG node removed here as it's not available in uboot yet